python/arcticdb/util/test.py

```diff
@@ -195,6 +197,14 @@ def configure_test_logger(level="INFO"):
     configure(get_test_logger_config(level=level, outputs=outputs), force=True)
 
 
+def gracefully_terminate_process(p):
+    p.terminate()
+    if sys.platform != "win32":
```

Comment:
What about windows?
Is sigkill considered as graceful termination seems rather violent to me?